### ISSUE TITLE
Paths table tooltips

### ISSUE THREAD

**Issue by dhimmel** [greenelab/hetmech-frontend#26]
Here are proposed tooltip texts for the Paths table:

+ metapath: same as in Metapaths table. Possibly with "Only paths corresponding to metapaths selected above are included."

+ path: sequence of edges in the network connecting from the source to the target node. Duplicate nodes are not permitted in paths.

+ path score: A score for how meaningful this path is in describing the connectivity between the source and target node. The score combines the magnitude of the metapath's p-value with the percent of the DWPC contributed by this path.

+ % of DWPC: The contribution of this path to the DWPC for its metapath. This metric compares the importance of all paths from the source to target node with the same metapath.

These probably need some touch up, but could do for now.

**Comment by vincerubinetti** [comment]
I've tweaked the wording a bit:

```json
{
  "path": "The sequence of edges in the network connecting the source node to the target node. Duplicate nodes are not permitted in paths.",
  "score": "A metric of how meaningful the path is in describing the connectivity betwen the source and target node. The score combines the magnitude of the metapath's p-value with the percent of the DWPC contributed by the path.",
  "percent_of_DWPC": "The contribution of the path to the DWPC for its metapath. This metric compares the importance of all paths of the same metapath from the source node to the target node."
}
```

The `metapath` field is already defined from the metapath table, so let's keep it the same. The tooltip names should exactly match the field names returned in the responses, as best as possible. Regarding adding "only the metapaths above are included", that information is already conveyed in other tooltips:

![image](https://user-images.githubusercontent.com/8326331/56767643-3c7ce580-677a-11e9-986f-b7ef7510577b.png)

![image](https://user-images.githubusercontent.com/8326331/56767658-456db700-677a-11e9-9709-37de78c7e5a5.png)

**Comment by dhimmel** [comment]
> I've tweaked the wording a bit:

Your rewording looks good to me.

**Comment by vincerubinetti** [comment]
Implemented in dev branch. Will be incorporated by future PR.

**Comment by vincerubinetti** [comment]
Closed by #41 #42 #43
